src/Copy/BrandCopy.elm

```diff
@@ -515,7 +515,7 @@ brandCopy key =
                 me which key things I wanted to remember to tell them."""
 
         StoryOne3ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a woman holding various stationery items"
```

Comment:
Would "Illustration of a happy woman holding a box of tissues and letter writing materials" work?